.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/faddat/cosmos
```

Comment:
can you link to this image? what does it contain?

Reply:
yes, the image is automatically built every hour here and I want to merge it into the cosmos-sdk-- the entire build system for it

https://github.com/faddat/archlinux-docker

The cosmos folder builds that image.

Reply:
if we merge:

https://github.com/cosmos/cosmos-sdk/pull/11768

Then that development environment image would be built here, hourly

Reply:
kk